sys/libraries/mailer.php

```diff
@@ -0,0 +1,54 @@
+<?php
+class mailer extends Library {
+
+	function __construct() {
+		$olddir = getcwd();
+		chdir(dirname(__FILE__));
+		require("PHPMailer/class.phpmailer.php");
+		//require('PHPMailer/PHPMailerAutoload.php');
+		chdir($olddir);
+		$this->mail = new PHPMailer;
+		$this->error = "";
+		$this->success = False;
+		$this->mail->isSMTP();
+		$this->mail->SMTPSecure = "tls";
+		$this->mail->Host = 'smtp.gmail.com';
+		$this->mail->Port = 587;
+		$this->mail->SMTPAuth = true;
+		$this->mail->Username = 'saathi.mailer@gmail.com';
+		$this->mail->Password = '';
+		$this->mail->FromName = "Saathi Mailer";
+		//	$this->mail->AddAddress("chandan.kharbanda@students.iiit.ac.in");                  // name is optional
+		//	$this->mail->AddAttachment("/var/tmp/file.tar.gz");         // add attachments
+		//	$this->mail->IsHTML(true);                                  // set email format to HTML
+		//	$this->mail->Subject = "Saathi emails";
+		//	$this->mail->Body    = "Welcome to <b>Saathi !</b>";
+		//	$this->mail->AltBody = "This is the body in plain text for non-HTML mail clients";
+	}
+
+	function getMail() {
+		return $this->mail;
+	}
+		
+	function sendMail() {
+		if(!$this->mail->Send())
+		{
+			$this->error = "Message could not be sent.";
+			$this->error .= "Mailer Error: " . $this->mail->ErrorInfo;
+			$this->success = False;
+		}
+		else
+		{
+			$this->error = "";
+			$this->success = true;
+		}
+		return $this->success;
+	}
+		
+	function showError() {
```

Comment:
Again, I don't see any need of this as we'll need custom error message every time. And also, we'll have to embed the message in views. 

It is not good practice to echo directly from library or controller or any place other than views.